src/components/resources/Resources.js

```diff
@@ -10,28 +10,28 @@ export default function Home() {
         Adopting pets is a much more humane way to find your familys newest best
         friend. Every year around 6.5 million companion pets enter shelters in
         just the USA, and thats only dogs and cats. Out of those
-        <b>6.5 million</b>, only <b>3.2 million</b> are adopted and
-        <b>1.5 million</b> are euthanized (Statistics provided by
+        <span> 6.5 million</span>, only <span>3.2 million</span> are adopted and
+        <span> 1.5 million</span> are euthanized (Statistics provided by
         <a
           target="_blank"
           rel="noreferrer"
           href="https://www.aspca.org/animal-homelessness/shelter-intake-and-surrender/pet-statistics"
         >
-          ASPCA
+          {""} ASPCA
         </a>
         ).
         <br />
-        <b>This leaves 1.8 million pets that go untouched each year.</b>
+        <span>This leaves 1.8 million pets that go untouched each year.</span>
         <br />
         <p>
           Adopting a pet is cheaper than buying a pet, and you also potentially
           save a life by doing so. Many shelter pets are already house-trained
           and are used to living with humans.{" "}
         </p>{" "}
         <br />
-        <b>
+        <span>
           More information about adopting pets can be found at the links below
-        </b>
+        </span>
         <br />
       </p>
```

Comment:
lets move this out paragraph outside of the p tag in line 9. That way it is not nested